pkg/model/insight.proto

```diff
@@ -77,15 +76,19 @@ message InsightDeployment {
   string app_id = 2;
   map<string,string> labels = 3;
 
+  // Time is the number of seconds since Unix epoch.
   int64 started_at = 4;
+  // Time is the number of seconds since Unix epoch.
   int64 completed_at = 5;
+  // Time is the number of seconds since Unix epoch.
   int64 rollback_started_at = 6;
-  model.DeploymentStatus complete_status = 7;
```

Comment:
Sorry for not making things clear 🙏  What I mean is to keep this `complete_status` and use type `model.DeploymentStatus` for this field. And besides, we should remove `InsightResultType` enum since we do not use it for now.

Reply:
My bad, looks like `InsightResultType` is used on web_service. Lets keep it as is 👌